### ISSUE TITLE
Farbschema

### ISSUE THREAD

**Issue by ClaudiaSchwegmann** [okfde/entwicklungsbarometer#40]
Im Drive neben den Wireframes habe ich zwei Farbschema vorgeschlagen English Edu und Feuerwehrauto. Wobei Feuerwehrauto mir wegen dem Grau besser gefällt - das würde meiner Meinung nach besser zu Schwarz-Weiß Bildern.

**Comment by milafrerichs** [comment]
![bildschirmfoto 2014-10-21 um 10 55 07](https://cloud.githubusercontent.com/assets/688980/4715593/07df47ba-5900-11e4-9d61-ed480cf1d734.png)
![bildschirmfoto 2014-10-21 um 10 41 37](https://cloud.githubusercontent.com/assets/688980/4715615/360a7b64-5900-11e4-8508-540a71ca6432.png)

**Comment by ClaudiaSchwegmann** [comment]
Ja!!! Das können wir erstmal so lassen und vielleicht noch von anderen okf
Kollegen ein paar Vorschläge bekommen.
Viele Grüsse
Claudia

Claudia Schwegmann
_Open Knowledge Foundation Germany_ http://okfn.de

Tel/Fax +49 (0) 5130 609691
Mobil +49 (0)177 3160 380 _NEW_
Twitter @OpenAidGermany
Skype Claudia.Schwegmann

_Die Open Knowledge Foundation setzt sich für offenes Wissen ein! Sie auch?
http://okfn.de/organisation/spenden/_

2014-10-21 11:02 GMT+02:00 Mila Frerichs notifications@github.com:
